api/src/main/java/com/pip/chatbot/ApplicationConfiguration.java

```diff
@@ -43,13 +46,38 @@ public CountriesRepository countriesRepository(DSLContext dsl) {
     }
 
     @Bean
-    public NoAuthJokeRepository noAuthJokeRepository(DSLContext dsl) {
-        return new NoAuthJokeRepository(dsl);
+    public JokesRepository noAuthJokeRepository(DSLContext dsl) {
+        return new JokesRepository(dsl);
     }
 
     @Bean
-    public MarkRepository markRepository(DSLContext dsl) {
-        return new MarkRepository(dsl);
+    @Qualifier("Joke")
+    public ModelMapper modelMapperJoke() {
```

Comment:
Miejmy jednego model mappera z konfiguracją wszystkich mapowań. Do sprawdzenia też czy potrzebujemy PropertyMap, kiedy propertasy się nazywają tak samo. Moim zdaniem to powinno zadziałać bez tych ustawień